data/mods/Tanks/items.json

```diff
@@ -422,7 +422,7 @@
   "sight_dispersion": 60,
   "recoil": -20,
   "durability": 8,
-  "clip_size": 1,
+  "clip_size": 5,
   "reload": 100,
   "loudness" : 300,
   "flags": [ "NO_RELOAD" ]
```

Comment:
This seems to be the way that "no reloading vehicle-only" weapons was enforced now that magazines have been implemented.

I suspect a development mistake, in that case?